src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformInterop.java

```diff
@@ -84,7 +84,7 @@ public static void setContentDescriptionForAutomationId(View view, String descri
 
     public static View getSemanticPlatformElement(View view) {
         if (view instanceof SearchView) {
-            view = view.findViewById(androidx.appcompat.R.id.search_button);
+            view = view.findViewById(androidx.appcompat.R.id.search_src_text);
```

Comment:
This seems like `search_src_text` is correct if this is meant to get the `TextView`:

https://github.com/androidx/androidx/blob/3078d9e8a726ae5f7a28d713cf4427e1bfc8e185/appcompat/appcompat/src/main/java/androidx/appcompat/widget/SearchView.java#L292

`search_button` is the button:

https://github.com/androidx/androidx/blob/3078d9e8a726ae5f7a28d713cf4427e1bfc8e185/appcompat/appcompat/src/main/java/androidx/appcompat/widget/SearchView.java#L298

Is this code trying to get the `TextView`?

Reply:
yep! We're trying to get the SearchView, and it looks like that TextView is being uesd as the SearchView.
The SearchButton stuff we were using before was wrong / not quite relevant

Reply:
Yea, this was initially setup before we changed SearchBar over to not just being the iconified version

https://github.com/dotnet/maui/commit/1d928baa35bdb7a8a9d7bfd9779ef977dc376693